deploy-service/common/src/main/java/com/pinterest/deployservice/events/DeployEvent.java

```diff
@@ -13,7 +13,7 @@ public class DeployEvent extends ResourceChangedEvent {
     private final String operator;
 
     public DeployEvent(Object source, String env, String stage, String commit, String operator) {
```

Comment:
Could potentially remove the `commit` param/field since it seems unused now

Reply:
Good point. Added logging so that information isn't lost